addons/mail/static/src/components/message_reaction_group_partner/message_reaction_group_partner.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.MessageReactionGroupPartner" owl="1">
+        <div class="o_MessageReactionGroupPartner" t-attf-class="{{ className }}" t-ref="root">
+            <div class="d-flex flex-column">
+                <t t-foreach="messageReactionGroupPartner.messageReactionGroup.partners" t-as='partner' t-key="partner">
+                    <div class="text-center m-3 d-flex">
+                        <img class="o_Follower_avatar me-2 rounded-circle" t-att-src="partner.avatarUrl" alt=""/>
+                        <t t-esc="partner.nameOrDisplayName" />
```

Comment:
avoid having a lonely text node as a sibling to another html tag. Always wrap it on an actual html node, e.g. `<span>`. This is important for proper styling, e.g. proper handling of text overflow such as with ellipsis